sagemaker/12_finetune_deploy_model/scripts/train.py

```diff
@@ -0,0 +1,99 @@
+"""
```

Comment:
Originally the tokenizer (but I see it has been since added to the original training script)
It also went through pylint which brought a few minor changes (import, naming, etc.)

Reply:
@philschmid I'll be the one maintaining this notebook.

Reply:
@philschmid - @cyranoB and I will maintain this notebook. We are working together on this for the blog post as well.